.github/workflows/basic.yml

```diff
@@ -11,15 +12,14 @@ permissions:
 
 jobs:
   icx-build:
-    name: Build - Intel C++ Compiler
+    name: Intel C++ Compiler
```

Comment:
can icx/icpx just be an include in ubuntu matrix? same as clang/clang++? I know this requires the use of a oneapi container for the whole matrix, but will this be a problem?

Reply:
I was thinking about it, but it would extend each Linux job execution by 2 minutes. These dockers are quite big. I would leave it for now, until we enable our own dockers.

Reply:
Done.